.github/workflows/ux-governance.yml

```
@@ -0,0 +1,33 @@
+
+      - name: Install dependencies
+        run: |
+          npm install
+
+      - name: Run UX Governance Check
+        run: |
+          node scripts/ux-ci-enforcer.js
+
+      - name: Upload UX Governance Report
+        if: failure()
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.9)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)